cmd/sops/main.go

```diff
@@ -855,6 +908,8 @@ func keyGroups(c *cli.Context, file string) ([]sops.KeyGroup, error) {
 	group = append(group, cloudKmsKeys...)
 	group = append(group, azkvKeys...)
 	group = append(group, pgpKeys...)
+	group = append(group, vaultMkKeys...)
+	log.Debugln("Master keys available: ", group)
```

Comment:
Nit but I'm seeing logs like `[CMD]	 DEBU[0000] Master keys available:  [0xc0002256e0]` from this line. Perhaps we use something like `log.Formatf("Master keys available: %+v", group)` to print its human readable contents instead of the address?